core/src/com/unciv/models/metadata/GameSettings.kt

```diff
@@ -341,18 +341,21 @@ class GameSettings {
         var autoPlayDiplomacy: Boolean = true
 
         var turnsToAutoPlay: Int = 0
+        var autoPlaying: Boolean = false
```

Comment:
Is there a need for this? Isn't it false only when turnsToAutoPlay is zero?

Reply:
Yes, AutoPlayEndTurn and maybe in the future AutoPlay Military want to say that they are AutoPlaying to change the text on nextTurnButton without activating the multiturn AutoPlay.

Reply:
That's *super* confusing.
Let's define what exactly we mean by "is autoplaying" and use terms accordingly- if this is only used for displaying different text when we're not really autoplaying then we should handle this differently

Reply:
We have 2 kinds of autoplay - one which*passes the turn* and one that does actions*within a turn* automatically? Let's make that clear